test/unittests/t_catalog_manager.cc

```diff
@@ -1,4 +1,4 @@
-/**
+/*
```

Comment:
No, in the last PR I already created only the skeleton of the file, but nothing else. However that extra star at the beginning of the file is a small error that is fixed in this PR